cmd/csidriver/main.go

```diff
@@ -87,8 +88,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err := fs.MkdirAll(filepath.Join(csiOpts.RootDir, dtcsi.GarbageCollectionPath), 0770); err != nil {
-		log.Error(err, "unable to create garbage collector directory for CSI Driver")
+	access := metadata.SqliteAccess{}
```

Comment:
Not using the constructor for this seems odd. As already pointed out, the initialization (including the setup step) of this object should be revised.